### DISCUSSION TITLE
Highlighter in table, how to make it case insensitive?

### DISCUSSION THREAD

**Discussion by Schouwenburg** [Textualize/rich#2840] [Q&A] [answered]
In my use case, I want to search for records in a database matching a search string. The output is presented as a rich table showing the matching records, and I want to highlight the text where it matches the search string. This all works, except that it now only highlights case sensitive. In my class for the RegexHighlighter I have the following regex definition. 

```
    class TextHighlighter(RegexHighlighter):
        highlights = [r"(?P<text>"+searchterm+"+)"]
```
Is there a way to highlight in a case-insensitive manner?

**Comment by willmcgugan** [comment, marked as answer]
Try prefixing with “(?i)”

**Reply by Schouwenburg** [reply]
Thank you! I was playing around with "i" already but could not find the right way.
